app/src/test/java/com/github/h3lp3rs/h3lp/MockForumTest.kt

```diff
@@ -60,8 +60,39 @@ class MockForumTest {
             .join()
     }
 
-    // TODO: Check listeners (way out of time), so their mocking stays risky and bug prone.
-    // TODO: Will do in the next sprint, the other functions have same the behaviour than fire forum
+    @Test
+    fun listenerWorksWhenPostsBefore() {
+        var counter = 0
+        forum.newPost(AUTHOR, CONTENT, isPost = true).orTimeout(TIMEOUT, MILLISECONDS)
+            .exceptionally { fail(TIMEOUT_FAIL_MSG) }.join()
+        forum.newPost(AUTHOR, CONTENT, isPost = true).orTimeout(TIMEOUT, MILLISECONDS)
+            .exceptionally { fail(TIMEOUT_FAIL_MSG) }.join()
+        forum.listenToAll { counter++ }
+        assertEquals(2, counter)
+    }
+
+    @Test
+    fun listenerWorksWhenPostAfter() {

```

Comment:
To be consistent with the method above (and to achieve my daily quota of nitpicking :))
```suggestion
    fun listenerWorksWhenPostsAfter() {

```